dissect/target/filesystems/tar.py

```diff
@@ -33,15 +33,20 @@ class TarFilesystem(Filesystem):
     def __init__(
         self,
         fh: BinaryIO,
-        base: Optional[str] = None,
-        tarinfo: Optional[tarfile.TarInfo] = None,
+        base: str | None = None,
+        tarinfo: tarfile.TarInfo | None = None,
+        _tarfile: tarfile.TarFile | None = None,
```

Comment:
```suggestion
        tarfile: tarfile.TarFile | None = None,
```
Don't specifically see a reason to prefix this with an underscore. Can you also add a docstring to this class that explains the arguments?

Reply:
Can change that into `import tarfile as tf` or something similar?